app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -158,6 +159,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = ""
     )
 
+    var currentWallpaper by stringPreference(
+        appContext.getPreferenceKey(R.string.pref_key_current_wallpaper),
+        default = ""
```

Comment:
Would it make sense for the default to be `Wallpapers.NONE`?

Reply:
Sure!